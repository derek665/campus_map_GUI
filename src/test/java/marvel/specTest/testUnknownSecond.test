# test correct output when second character is unknown

LoadGraph testSecond soccer.tsv

FindPath testSecond Liverpool Bilbao