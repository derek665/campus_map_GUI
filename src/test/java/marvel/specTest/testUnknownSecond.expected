# test correct output when second character is unknown

loaded graph testSecond

unknown character Bilbao